### PR TITLE
Dram estimates

### PR DESCRIPTION
**Pull request by olly-larkin**

### CODE REVIEW

File: Simulator/python/src/Estimator.py [hunk 1]
```diff
@@ -67,26 +112,82 @@
...
+    },
+    "MEMCI_bidir2_tss": {
+        'name': 'MEMCI_bidir2_tss',
+        'description': 'MEMCI, Bidirectional 2, Three Step Search',
+        'options': default_tss_options
+    },
+    "MEMCI_bidir2_hbma": {
+        'name': 'MEMCI_bidir2_hbma',
+        'description': 'MEMCI, Bidirectional 2, Hierarchical Block Matching Algorithm',
+        'options': default_hbma_options
```
Comment (lhl2617) [inline Simulator/python/src/Estimator.py:190]: Has these been tested from CLI + gui?

Comment (lhl2617) [reply Simulator/python/src/Estimator.py:190]: have*

Comment (olly-larkin) [reply Simulator/python/src/Estimator.py:190]: Cli works but I have no idea what's going on with the gui I haven't looked

**Comment by lhl2617** [comment]
Looks good to me, approved if confirmed GUI working

**Comment by lhl2617** [comment]
Should we add answers to Bouganis's "which board does this target?"

File: Simulator/python/src/Estimator.py (hunk 1 above)
Comment (lhl2617) [reply Simulator/python/src/Estimator.py:190]: I think @NGovani  is trying?

Comment (NGovani) [reply Simulator/python/src/Estimator.py:190]: Yeah will check today, just need to build the gui on our branch

**Comment by NGovani** [comment]
Nah, we only use the board as a way to see if the schemes we use are feasible, The calculations dont change depending on the target board

**Comment by lhl2617** [comment]
If you follow the steps for Debug you can test it against fresh code on your branch. I can built it tonight

**Comment by olly-larkin** [comment]
ok its all working

GUI tested

**Comment by lhl2617** [comment]
Ok approved. I will build 1.0.0 tonight, test, and release

**Pull request merged**